src/learner.cc

```diff
@@ -97,9 +101,39 @@ struct LearnerModelParam : public dmlc::Parameter<LearnerModelParam> {
   }
 };
 
+struct LearnerTrainParam : public dmlc::Parameter<LearnerTrainParam> {
+  // data split mode, can be row, col, or none.
+  DataSplitMode dsplit;
+  // flag to disable default metric
+  int disable_default_eval_metric;
+
+  std::string booster;
+  std::string objective;
+
+  // declare parameters
+  DMLC_DECLARE_PARAMETER(LearnerTrainParam) {
+    DMLC_DECLARE_FIELD(dsplit)
```

Comment:
I don't think this parameter is even supported any more as there is no distributed column training. I wonder if we should remove it and reimplement if we ever decide to bring back distributed column training. So much has changed since this was used.

Reply:
Let me think about it later.